docs/api/ref/DesktopAgent.md

```diff
@@ -1476,7 +1476,7 @@ Not implemented
 </TabItem>
 </Tabs>
 
-Adds a listener for incoming context broadcasts from the Desktop Agent. Provided for backwards compatibility with versions FDC3 standard <2.0.
+Adds a listener for incoming context broadcasts from the Desktop Agent. Provided for backwards compatibility with versions FDC3 standard\<2.0.
```

Comment:
The escape in all these instances replaced the space so resulting format is now "standard<2.0"

Reply:
```suggestion
Adds a listener for incoming context broadcasts from the Desktop Agent. Provided for backwards compatibility with versions FDC3 standard &lt;2.0.
```